Small changes

yiwan-rl opened the pull request.

btanner reviewed (approved):
This PR looks good to me. I plan to setup some automated testing in the future, but for now I'm happy to merge this.

btanner commented:
@yiwan-rl I know it's a hassle, but ideally it would be great if you could add some text to explain the changes.

- Removes "src." prefix from some imports
- Updates some hyper params in the notebook
- etc

yiwan-rl commented:
@btanner I removed src. prefix because your last PR also removed that. I thought it was intentional. The hyperparameters that you mentioned are not hyperparameters of the experiments, I think they are outputs generated by running the ipython notebook. I don't know how to change or remove these outputs. So I just kept them.
